specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Operations.json

```diff
@@ -1,175 +0,0 @@
-{
-    "swagger": "2.0",
```

Comment:
Why this file is removed?

Reply:
This is where the long running operations poll.  They have no way to call it unless they do some packet sniffing to get the operation id.